src/templates/publication.js

```diff
@@ -107,6 +108,7 @@ const Render = ({ data }) => {
 	<center>{coverImage}</center>
 	<br/>
         {citeLink}
+    <Citation publication={publication}/>
```

Comment:
Can this go after the abstract, on one line, *Export citation: <dropdown> EXPORT*?

Reply:
Done